var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -60,12 +62,16 @@ class Gromacs(CMakePackage):
     version('4.6.7', sha256='6afb1837e363192043de34b188ca3cf83db6bd189601f2001a1fc5b0b2a214d9')
     version('4.5.5', sha256='e0605e4810b0d552a8761fef5540c545beeaf85893f4a6e21df9905a33f871ba')
 
-    variant('mpi', default=True, description='Activate MPI support (disable for Thread-MPI support)')
+    variant('parallel', default='thread-mpi,mpi',
+            values=('thread-mpi', 'mpi'),
+            multi=True,
+            description='parallelization mode (Thread-MPI or MPI support)')
     variant('shared', default=True,
             description='Enables the build of shared libraries')
-    variant(
-        'double', default=False,
-        description='Produces a double precision version of the executables')
+    variant('precision', default='single',
+            values=('single', 'double'),
```

Comment:
precision should probably have the option 'mixed' instead of 'single', since this is how Gromacs calls it since at least 2018

Reply:
I don't think that's a good idea because this is a multi-valued variant.

Reply:
I don't understand. My proposal was just to rename single to mixed to match the Gromacs terminology.

Reply:
Ok, I thought you meant mixed as equivalent to precision=single,double. I think it's only extra confusing when we allow multiple precision in one build.